packages/directory/tests/acceptance/dir-table-test.js

```diff
@@ -112,4 +112,52 @@ module('Acceptance | dir table', function (hooks) {
       .dom('.dir-table-filter select')
       .hasValue('Dashboards', 'The selected type is set based on the query param in the url');
   });
+
+  test('favorite item toggle', async function (assert) {
+    assert.expect(9);
+    await visit('/directory/my-data');
+    const user = await this.owner.lookup('service:user').findUser();
```

Comment:
Can we maybe replace the use of the user service with a function that returns a list of all favorite reports from the dom?

```
const getFavorites = () => {
	return findAll('...').filter(el => el.querySelector('.d-star-solid')).map(fav => fav.querySelector('.dir-item-name-cell').textContent.trim())
}